modules/simple_forms_api/app/form_mappings/vba_21_0845.json.erb

```diff
@@ -0,0 +1,93 @@
+  "F[0].Page_2[0].MailingAddress_ZIPOrPostalCode_LastFourNumbers[0]": "<%= data.dig('authorizer_address', 'postal_code')&.[](5..8) %>",
+  "F[0].Page_2[0].Telephone_Number_Last_Four_Numbers[0]": "<%= data['authorizer_phone']&.[](6..9) %>",
+  "F[0].Page_2[0].Telephone_Number_Second_Three_Numbers[0]": "<%= data['authorizer_phone']&.[](3..5) %>",
+  "F[0].Page_2[0].Telephone_Number_Area_Code[0]": "<%= data['authorizer_phone']&.[](0..2) %>",
+  "F[0].Page_2[0].International_Telephone_Number_If_Applicable[0]": "",
+  "F[0].Page_2[0].CheckBox1[0]": "<%= data['authorizer_email_consent'] ? 1 : 0 %>",
+  "F[0].Page_2[0].Email_Address_Optional[0]": "<%= data['authorizer_email'] %>",
+
+  "F[0].Page_2[0].NAME_OF_AUTHORIZED_PERSON_First_Name[0]": "<%= data.dig('person_full_name', 'first') %>",
+  "F[0].Page_2[0].Authorized_Person_Middle_Initial1[0]": "<%= data.dig('person_full_name', 'middle') %>",
+  "F[0].Page_2[0].Authorized_Person_Last_Name[0]": "<%= data.dig('person_full_name', 'last') %>",
+  "F[0].Page_2[0].ADDRESS_OF_PERSON_Number_And_Street[0]": "<%= data.dig('person_address', 'street') %>",
+  "F[0].Page_2[0].ADDRESS_OF_PERSON_ApartmentOrUnitNumber[0]": "<%= data.dig('person_address', 'street2') %>",
+  "F[0].Page_2[0].ADDRESS_OF_PERSON_City[0]": "<%= data.dig('person_address', 'city') %>",
+  "F[0].Page_2[0].ADDRESS_OF_PERSON_StateOrProvince[0]": "<%= data.dig('person_address', 'state') %>",
+  "F[0].Page_2[0].ADDRESS_OF_PERSON_Country[0]": "<%= data.dig('person_address', 'country') %>",
+  "F[0].Page_2[0].ADDRESS_OF_PERSON_ZIPOrPostalCode_LastFourNumbers[0]": "<%= data.dig('person_address', 'postal_code')&.[](0..4) %>",
+  "F[0].Page_2[0].ADDRESS_OF_PERSON_ZIPOrPostalCode_FirstFiveNumbers[0]": "<%= data.dig('person_address', 'postal_code')&.[](5..8) %>",
+
+  "F[0].Page_2[0].International_Telephone_Number_If_Applicable[1]": "<%= data['organization_name']&.+('\n')&.+(data['organization_representatives']&.map { |rep| rep['representative_name'] }.join(', ')) %>",
+  
+  "F[0].#subform[2].VeteransSocialSecurityNumber_FirstThreeNumbers[0]": "<%= data['veteran_ssn']&.[](0..2) %>",
+  "F[0].#subform[2].VeteransSocialSecurityNumber_SecondTwoNumbers[0]": "<%= data['veteran_ssn']&.[](3..4) %>",
+  "F[0].#subform[2].VeteransSocialSecurityNumber_LastFourNumbers[0]": "<%= data['veteran_ssn']&.[](5..8) %>",
+
+  "F[0].#subform[2].ADDRESS_OF_ORGANIZATION_NumberAndStreet[0]": "<%= data.dig('organization_address', 'street') %>",
+  "F[0].#subform[2].ApartmentOrUnitNumber[0]": "<%= data.dig('organization_address', 'street2') %>",
+  "F[0].#subform[2].City[0]": "<%= data.dig('organization_address', 'city') %>",
+  "F[0].#subform[2].StateOrProvince[0]": "<%= data.dig('organization_address', 'state') %>",
+  "F[0].#subform[2].Country[0]": "<%= data.dig('organization_address', 'country') %>",
+  "F[0].#subform[2].ZIPOrPostalCode_LastFourNumbers[0]": "<%= data.dig('organization_address', 'postal_code')&.[](0..4) %>",
```

Comment:
are these 2 definitions of first and last correct? should this one be first instead of last?